bindings/java/c_kzg_4844_jni.c

```diff
@@ -38,7 +38,7 @@ void throw_c_kzg_exception(JNIEnv *env, C_KZG_RET error_code, const char *messag
 void throw_invalid_size_exception(JNIEnv *env, const char *prefix, size_t size, size_t expected_size)
 {
   char message[100];
```

Comment:
For reference: why is this 100?

Reply:
I believe it's an arbitrary length that's enough for all of the current messages.

cc @StefanBratanov

Reply:
Sounds good -- If messages grow beyond 100 characters, it will truncate so I think its fine regardless.

Reply:
Agreed. Thanks for the review 🥇 will merge.

Reply:
Yeah pretty much what Justin said, wasn't sure the best way to handle this

Reply:
It's all good, that's how I would do it too. We can always raise that value later if we need to.